certified-connectors/Mural/apiProperties.json

```diff
@@ -0,0 +1,41 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ client id }}",
+          "scopes": [
+            "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/"
+            },
+            "tokenUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/refresh"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
+    "scriptOperations": [
+      "CreateNewStickyNote"
+    ],
+    "capabilities": [],
+    "policyTemplateInstances": [],
```

Comment:
@dmytroIvanovDev If your connector has no policy templates, please remove this property. Empty property values can be removed until unless you have a specific reason to keep them.